.github/workflows/stage-deploy.yml

```diff
@@ -41,3 +52,17 @@ jobs:
 
       - name: Deploy pages
         run: aws s3 sync docs/ s3://interactive-examples-stage-ce3e7ea385c68f44/
+
+      - name: Authenticate with GCP
+        uses: google-github-actions/auth@v0
+        with:
+          token_format: access_token
+          service_account:  deploy-stage-interactive-examp@{{ secrets.GCP_PROJECT_NAME }}.iam.gserviceaccount.com
+          workload_identity_provider: projects/${{ secrets.WIP_PROJECT_ID }}/locations/global/workloadIdentityPools/github-actions/providers/github-actions
+
+      - name: Setup gcloud
+        uses: google-github-actions/setup-gcloud@v1
+
+      - name: Deploy pages to GCP
+        run: |-
+          gsutil -m -h "Cache-Control:public, max-age=86400" rsync -d -r docs/ gs://interactive-examples-stage-mdn/
```

Comment:
Currently, we deploy `interactive-examples` as soon as a PR gets merged. If I understand this correctly, this would prevent changes from a PR that is merged within 24h of another PR to be immediately visible. Do we need another step to invalidate the CDN?

Reply:
This sets the cache-control on the object, so it would be set once the CDN hits it. I can remove though and we can always set again if needed.